lgc/patch/NggPrimShader.cpp

```diff
@@ -4191,8 +4191,13 @@ Function *NggPrimShader::createBackfaceCuller(Module *module) {
 
     // threeshold = (10 ^ (-backfaceExponent)) / |w0 * w1 * w2|
     auto threshold = m_builder->CreateNeg(backfaceExponent);
+#if LLVM_MAIN_REVISION < 391319
```

Comment:
Isn't there a convention to write `#if LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 391319` so that if LLVM_MAIN_REVISION is undefined it assumes you're using a bang up-to-date LLVM?

Reply:
Yes - I didn't cut and paste it from the template like I should have (I just like typing).
I'll fix it.